.eslintrc.yml

```diff
@@ -0,0 +1,11 @@
+env:
+  browser: true
+  commonjs: true
+  es2021: true
+  mocha: true
+extends: "eslint:recommended"
+parserOptions:
+  ecmaVersion: 12
+rules: {}
+globals:
+  "__dirname": readonly
```

Comment:
This should be able to go away if the `node` environment is set

Reply:
indeed !